core/primitives/src/views.rs

```diff
@@ -648,7 +648,7 @@ pub struct ChunkHeaderView {
     pub encoded_length: u64,
     pub height_created: BlockHeight,
     pub height_included: BlockHeight,
-    pub shard_id: ShardId,
+    pub shard_id: ShardOrd,
```

Comment:
I think it is better to add a new field called `shard_ord` that is synonymous to what `shard_id` means here and keep both. Later on we can deprecate `shard_id`. @frol what do you think?

Reply:
See my comment above. I assume we won't move to `shard_ord` and rather hide it, so `shard_id` is the right naming.